mkt/lookup/templates/lookup/includes/app_search.html

```diff
@@ -5,6 +5,7 @@
       {{ _('App Lookup') }}
     </label>
     <input type="search" id="app-search" name="q" autocomplete="off">
+    <input type="hidden" id="app-lang" name="lang" autocomplete="off" value="en-US">
```

Comment:
It's accessible at `window.navigator.l10n.language`, if the lookup tool uses the same setup as commonplace.